src/main/java/org/opentripplanner/graph_builder/module/osm/AreaGroup.java

```diff
@@ -162,6 +162,9 @@ private Ring toRing(Polygon polygon, HashMap<Coordinate, OSMNode> nodeMap) {
     }
 
     public OSMWithTags getSomeOSMObject() {
+        if (!areas.iterator().hasNext()) {
+            throw new IllegalArgumentException("No OSM object available.");
+        }
         return areas.iterator().next().parent;
```

Comment:
This create an extra iterator and does not provide much.
I think we can close this, I don´t think it is an improvement.